### PR TITLE
1.3 forgot password Kolisnyk

### PR DESCRIPTION
**Pull request by kolesnicknick**

### CODE REVIEW

**Review by so2niko — APPROVED** [review]
ок

**Review by OksanaKozhukh — APPROVED** [review]

File: front/src/common/redux/forgot-password/forgot.password.api.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export async function forgotPassswordApi (request) {
```
Comment (Alan18081) [inline front/src/common/redux/forgot-password/forgot.password.api.js]: Please, don't pass actual action, pass only data. API function should not know anything about redux

File: front/src/common/redux/forgot-password/forgot.password.api.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    const url = `${process.env.REACT_APP_FORGOT_PASSWORD_ROUTE}`;
+    const headers = { headers: { 'Content-Type': 'application/json' } };
+
+    return axios
+        .post(url, request.data, headers)
+        .then(response => response)
```
Comment (Alan18081) [inline front/src/common/redux/forgot-password/forgot.password.api.js]: API function should return only the actual data. Saga should know nothing about any HTTP related data

File: front/src/common/redux/forgot-password/forgot.password.action.js [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import { FORGOT_PASSWORD } from './types';
+
+export const forgotPasswordAction = (data) => ({
+    type: FORGOT_PASSWORD,
+    data
```
Comment (Alan18081) [inline front/src/common/redux/forgot-password/forgot.password.action.js]: Please, use `payload` key instead of `data`

Comment (kolesnicknick) [reply front/src/common/redux/forgot-password/forgot.password.action.js]: done

File: front/src/common/redux/forgot-password/forgot.password.api.js (hunk 2 above)
Comment (kolesnicknick) [reply front/src/common/redux/forgot-password/forgot.password.api.js]: done

File: front/src/common/redux/forgot-password/forgot.password.api.js (hunk 1 above)
Comment (kolesnicknick) [reply front/src/common/redux/forgot-password/forgot.password.api.js]: done

**Review by Alan18081 — APPROVED** [review]

**Pull request merged**